YoutubeDownloader/Framework/DialogManager.cs

```diff
@@ -68,7 +68,7 @@ await DialogHost.Show(
         return file?.TryGetLocalPath() ?? file?.Path.ToString();
     }
 
-    public async Task<string?> PromptDirectoryPathAsync(string defaultDirPath = "")
+    public static async Task<string?> PromptDirectoryPathAsync(string defaultDirPath = "")
```

Comment:
```suggestion
    public async Task<string?> PromptDirectoryPathAsync(string defaultDirPath = "")
```

Please avoid unrelated changes